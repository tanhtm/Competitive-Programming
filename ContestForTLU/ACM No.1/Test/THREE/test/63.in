355549050000000000
