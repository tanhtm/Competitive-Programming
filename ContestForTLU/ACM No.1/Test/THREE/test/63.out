4 5 7 10 11 15 19 23 24 26 28 30 36 
2 3 6 13 16 20 21 25 33 35 37 
